6.Database/Project/CRUD.js

```diff
@@ -0,0 +1,72 @@
+const mysql = require('mysql2')
+
+const pool = mysql.createPool({             // Making connection with database
+    host:'localhost',
```

Comment:
Please use .env file for DB or any secrets

Reply:
Updated